donate/models.py

```diff
@@ -253,8 +253,8 @@ class StripeSubscription(db.Model, TimestampMixin):
     # user = db.Column(db.Integer,
     #                  db.ForeignKey('user.id'),
     #                  nullable=False)
-    email = db.Column(db.String)
-    customer_id = db.Column(db.String)
+    email = db.Column(db.String(128))
```

Comment:
Based on [this stackoverflow answer](https://stackoverflow.com/a/574698), the maximum length of an email address is, in practice, 254 characters so I think that would be a better length to choose here.